app/models/foreman_tasks/task.rb

```diff
@@ -123,6 +123,10 @@ def pending?
     def resumable?
       false
     end
+    
+    def cancellable?
+      execution_plan&.cancellable?
+    end
```

Comment:
This is already defined on `ForemanTasks::Task::DynflowTask` and should not be needed.

Reply:
Removed, thanks!